components/molecule/inputTags/src/index.scss

```
@@ -20,6 +20,10 @@ $sizes-atom-input-paddings: m $p-m, s $p-s !default;
     @extend %sui-atom-input-input-focus;
   }
 
+  &--disabled {
+    background-color: $c-gray-light-4;
```

IMO, I would define a component variable

```
$bg-input-tag-disable: $c-gray-light-4 !default;
```

```suggestion
    background-color: $bg-input-tag-disable;
```